src/frontend/flask_server.py

```diff
@@ -104,6 +106,18 @@ def home():
                            message=request.args.get('msg', None))
 
 
+def _populate_contact_labels(account_id, transactions, contacts):
```

Comment:
might be helpful to better document this

Reply:
Good point, it is a bit complicated =)